app/views/layouts/_header.html.haml

```diff
@@ -12,8 +12,10 @@
   %nav.collapse.navbar-collapse
     %ul.nav.navbar-nav.navbar-right.navbar-iconic
       %li{:class => "drawer-pf-trigger dropdown"}
-        %a{:class => "nav-item-iconic drawer-pf-trigger-icon", "ng-click" => "vm.toggleNotificationsList()"}
-          %span{:title => _("Notifications"), "ng-class" => "{'fa fa-bell': vm.newNotifications, 'fa fa-bell-o': !vm.newNotifications}"}
+        %a{:class => "nav-item-iconic drawer-pf-trigger-icon", :title => "{{vm.notificationsIndicatorTooltip}}", "ng-click" => "vm.toggleNotificationsList()"}
+          %span{"ng-class" => "{'fa fa-bell': vm.newNotifications, 'fa fa-bell-o': !vm.newNotifications}"}
+          %span.badge.info{"ng-show" => "vm.newNotifications"}
+            &#8226
```

Comment:
Isn't there a fonticon with that code? Or something semantic, `8226` doesn't really tell us much .. possibly just add a comment what it is?